### PR TITLE
New requirements

### PR DESCRIPTION
**Pull request by cesaramaya-flugel**

### CODE REVIEW

File: packer.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
+{
+  "builders": [{
+		"ami_name": "flugel-test",
+		"instance_type": "t2.micro",
+		"region": "us-east-1",
+		"type": "amazon-ebs",
+		"source_ami": "ami-039a49e70ea773ffc",
+		"ssh_username": "ubuntu"
+	}],
+	"provisioners": [{
```
Comment (diegows) [inline packer.json:10]: I don't see where are you installing traefik now with packer.

Comment (cesaramaya-flugel) [reply packer.json:10]: For me is better to install Traefik along with the S3-Proxy at launch time. But if this a must I can have it installed in the image build.

**Comment by diegows** [comment]
Yes, if we use packer we expect the image to be immutable.

**Comment by cesaramaya-flugel** [comment]
Got it.

I'm working on it.
ᐧ

**Pull request merged**